lib/Models/WebMapServiceCatalogItem.js

```diff
@@ -131,6 +132,21 @@ var WebMapServiceCatalogItem = function(terria) {
      */
     this.minScaleDenominator = undefined;
 
+    /**
+     * Gets or sets the denominator of the largest scale (smallest denominator) beyond which to show a message explaining that no further zoom levels are available, at the request
+     * of the data custodian.
+     * @type {Number}
+     */
+    this.maximumScaleBeforeMessage = undefined;
```

Comment:
Should I change the `maximumScaleToLevel` function to `minScaleDenominatorToLevel` too (if this function is even necessary)?

Reply:
Maybe just `scaleDenominatorToLevel`? This is less important to get right, though, because it's a private function. Users won't notice or care if we change its name later.